docs/se/reactivemessaging/introduction.adoc

```diff
@@ -121,7 +121,7 @@ implicit ack before callback is executed is necessary.
 
 === Connector
 Connector concept is a way for connecting <<Channel,channels>> to external sources.
```

Comment:
Connector concept? Does this mean the concept of connecting channels to external sources or could it be "Connectors are used to connect channels to external sources."?